src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/ContentDbContextModelSnapshot.cs

```diff
@@ -1019,6 +1050,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .WithOne()
                         .HasForeignKey("GovUk.Education.ExploreEducationStatistics.Content.Model.DataImport", "ZipFileId")
                         .OnDelete(DeleteBehavior.Restrict);
+
+                    b.Navigation("File");
```

Comment:
I'm guessing as with other misc updates in this file, this is as a result of trying to create our first Migration since the .NET / EF upgrade?

Reply:
Since updating my own CLI tools I think. A new migration after the update but with the old version of the EF tools CLI wouldn't have these changes I believe.

You can update with `dotnet tool update --global dotnet-ef`